tests/snappi_tests/multidut/bgp/files/bgp_outbound_helper.py

```diff
@@ -0,0 +1,1185 @@
+        ipv4.name = 'T2_IPv4_%d' % lag_count
+        ipv4.address = snappi_portchannel_ipv4_list[lag_count]
+        ipv4.gateway = t2_dut_portchannel_ipv4_list[lag_count]
+        ipv4.prefix = v4_prefix_length
+        ipv6 = eth.ipv6_addresses.add()
+        ipv6.name = 'T2_IPv6_%d' % lag_count
+        ipv6.address = snappi_portchannel_ipv6_list[lag_count]
+        ipv6.gateway = t2_dut_portchannel_ipv6_list[lag_count]
+        ipv6.prefix = v6_prefix_length
+
+        bgpv4 = device.bgp
+        bgpv4.router_id = t2_dut_portchannel_ipv4_list[lag_count]
+        bgpv4_int = bgpv4.ipv4_interfaces.add()
+        bgpv4_int.ipv4_name = ipv4.name
+        bgpv4_peer = bgpv4_int.peers.add()
+        bgpv4_peer.name = 'T2_BGP_%d' % lag_count
+        bgpv4_peer.as_type = BGP_TYPE
+        bgpv4_peer.peer_address = t2_dut_portchannel_ipv4_list[lag_count]
+        bgpv4_peer.as_number = int(T2_SNAPPI_AS_NUM)
+
+        route_range1 = bgpv4_peer.v4_routes.add(name="T2_IPv4_Routes_%d" % (lag_count))
+        for route_index, routes in enumerate(route_range['IPv4']):
+            route_range1.addresses.add(
+                address=routes[0], prefix=routes[1], count=routes[2])
+        for community in snappi_community_for_t2:
+            manual_as_community = route_range1.communities.add()
+            manual_as_community.type = manual_as_community.MANUAL_AS_NUMBER
+            manual_as_community.as_number = int(community.split(":")[0])
+            manual_as_community.as_custom = int(community.split(":")[1])
+        ipv4_dest.append(route_range1.name)
+
+        bgpv6 = device.bgp
+        bgpv6.router_id = t2_dut_portchannel_ipv4_list[lag_count]
+        bgpv6_int = bgpv6.ipv6_interfaces.add()
+        bgpv6_int.ipv6_name = ipv6.name
+        bgpv6_peer = bgpv6_int.peers.add()
+        bgpv6_peer.name = 'T2_BGP+_%d' % lag_count
+        bgpv6_peer.as_type = BGP_TYPE
+        bgpv6_peer.peer_address = t2_dut_portchannel_ipv6_list[lag_count]
+        bgpv6_peer.as_number = int(T2_SNAPPI_AS_NUM)
+
+        route_range2 = bgpv6_peer.v6_routes.add(name="T2_IPv6_Routes_%d" % (lag_count))
+        for route_index, routes in enumerate(route_range['IPv6']):
+            route_range2.addresses.add(
+                address=routes[0], prefix=routes[1], count=routes[2])
+        for community in snappi_community_for_t2:
+            manual_as_community = route_range2.communities.add()
+            manual_as_community.type = manual_as_community.MANUAL_AS_NUMBER
+            manual_as_community.as_number = int(community.split(":")[0])
+            manual_as_community.as_custom = int(community.split(":")[1])
+        ipv6_dest.append(route_range2.name)
+
+    for index, port in enumerate(snappi_t1_ports):
+        if len(str(hex(index+1).split('0x')[1])) == 1:
+            m = '0'+hex(index+1).split('0x')[1]
+        else:
+            m = hex(index+1).split('0x')[1]
+
+        device = config.devices.device(name="T1 Device {}".format(index))[-1]
```

Comment:
Can we change it to following:

Tx Port side: Change it to T0 Device

Other links: Backup T2_%

Reply:
implemented